ketchupbot-updater/API/MwClient.cs

```diff
@@ -16,6 +19,17 @@ public class MwClient
         CookieContainer = new CookieContainer()
     });
 
+    private static readonly HttpClient Client2 =
+        new(new PolicyHttpMessageHandler(HttpPolicyExtensions.HandleTransientHttpError().WaitAndRetryAsync(5,
+            retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt))))
+        {
+            InnerHandler = new HttpClientHandler
+            {
+                UseCookies = true,
+                CookieContainer = new CookieContainer()
+            }
+        });
+
```

Comment:
ok whoops this is code duplication here (client2 instead of overwriting client). prob gonna have to clean that up